scripts/nb-tester/pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[build-system]
+requires = [ "hatchling",]
+build-backend = "hatchling.build"
+
+[project]
+name = "qiskit-docs-notebook-tester"
```

Comment:
Nit: `ibm-quantum-notebook-tester` is more appropriate.

Reply:
I'm not sure, the package is specific to our team and I think this name makes it clear where the package comes from (e.g. if someone's inspecting their environment).